lib/hammer_cli_foreman/personal_access_tokens.rb

```diff
@@ -0,0 +1,46 @@
+module HammerCLIForeman
+  class PersonalAccessTokens < HammerCLIForeman::Command
+    resource :personal_access_tokens
+    command_name "access-token"
+    desc _("Manage Personal Access Tokens")
+
+    class ListCommand < HammerCLIForeman::ListCommand
+      output do
+        field :id, _("Id")
+        field :name, _("Name")
+        field :active, _("Active"), Fields::Boolean
```

Comment:
This always showed 'no' in my testing even though the token was `:active => true`